.ci_support/linux_64_.yaml

```diff
@@ -1,7 +1,7 @@
 cdt_name:
 - cos6
 channel_sources:
-- conda-forge/label/bw2data_dev,conda-forge,defaults
```

Comment:
Should be `conda-forge/label/bw2calc_dev,conda-forge/label/bw2calc_dev,conda-forge` as `0.9dev22` depends on both development versions.